app/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.material:material:1.0.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
What have you used this implementation for?

Reply:
I did not add it explicitly, it might have been automatically added. Since the Fragment class uses androidx. "import androidx.fragment.app.Fragment" This is the auto generated import statement.